sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2405,8 +2407,7 @@ def teardown_no_lock(self,
                     stream_logs=False,
                     require_outputs=True)
         elif (terminate and
-              (prev_status == global_user_state.ClusterStatus.STOPPED or
-               use_tpu_vm)):
```

Comment:
Is this part the cause of the bug you mean by `This PR also fixes the error of sky status --all after user manually terminates a TPU VM instance`?

Reply:
The reason I remove is `ray down` does work for multi-node TPU VMs.

If we use `gcloud` to teardown the multi-node TPU VMs, we have to do additional step to stop the ray cluster first. otherwise, ray autoscaler will launch the worker VM again. does that make sense?

Reply:
Sounds great!